web/src/App.tsx

```diff
@@ -1,18 +1,25 @@
 import { Provider as Auth0Provider } from "@reearth-cms/auth";
 import NotFound from "@reearth-cms/components/atoms/NotFound";
+import Account from "@reearth-cms/components/organisms/Settings/Account";
 import DashboardPage from "@reearth-cms/components/pages/Dashboard";
 import RootPage from "@reearth-cms/components/pages/RootPage";
+import WorkspacePage from "@reearth-cms/components/pages/Workspace";
 import { Provider as GqlProvider } from "@reearth-cms/gql";
 import { Provider as I18nProvider } from "@reearth-cms/i18n";
 import { BrowserRouter as Router, useRoutes } from "react-router-dom";
 
 import "./App.css";
 import "antd/dist/antd.css";
+import WorkspaceListPage from "./components/pages/WorkspaceList";
```

Comment:
Can you move this up where it seems more appropriate?

Reply:
We should actually setup import filtering like we do in Re:Earth so we don't have to worry about the order everytime